whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/Inet6numStatusChildSpec.groovy

```diff
@@ -1863,11 +1863,12 @@ class Inet6numStatusChildSpec extends BaseQueryUpdateSpec {
         ack.summary.assertSuccess(0, 0, 0, 0, 0)
         ack.summary.assertErrors(1, 1, 0, 0)
 
-        ack.countErrorWarnInfo(1, 0, 0)
+        ack.countErrorWarnInfo(2, 0, 0)
         ack.errors.any { it.operation == "Create" && it.key == "[inet6num] 2001:600::/32" }
         ack.errorMessagesFor("Create", "[inet6num] 2001:600::/32") ==
-                ["\"assignment-size:\" attribute only allowed with status AGGREGATED-BY-LIR"]
-
+                ["\"assignment-size:\" attribute only allowed with status AGGREGATED-BY-LIR",
+                 "\"mnt-lower:\" attribute not allowed for resources with \"ASSIGNED:\" status"
```

Comment:
Why is the new error showing up in an unrelated test?

Reply:
Remove "mnt-lower:" from test data in ""create child ASSIGNED, with assignment-size" test

Reply:
Done